src/internal/connector/onedrive/collections_test.go

```diff
@@ -1244,11 +1283,14 @@ func (suite *OneDriveCollectionsSuite) TestGet() {
 							driveItem("folder", "folder", driveBasePath1, "root", false, true, false),
 							driveItem("file", "file", driveBasePath1+"/folder", "folder", true, false, false),
 						},
-						deltaLink: &empty,
+						deltaLink: &empty, // not sure if this is a valid scenario
```

Comment:
fair point, it's probably not. I think I probably added it cause there's a check in other code to only persist delta token stuff if `len(deltaLink) > 0` 😅

Reply:
I'll keep the comment there just for reference.